pandas/tests/tseries/test_offsets.py

```diff
@@ -502,6 +502,31 @@ def test_pickle_v0_15_2(self):
         #
         tm.assert_dict_equal(offsets, read_pickle(pickle_path))
 
+    def test_pickle_v0_20_3(self):
```

Comment:
none of this is needed

Reply:
these go in the generate_legacy_storage_files script instead. (which you then use to generate). I think the instructions are now pretty clear. If they are not, feel free to update them as well (at the top of that file).

Reply:
OK, just removed that test.  As to the instructions, I'm still unclear on where the actual _test_ goes and what it looks like.

Reply:
the test is generic it runs ALL of the pickle files

that’s the point you don’t actually need to construct a test just put a new pickle file

Reply:
OK.  `test_pickle_v0_20_3` has been removed, so it sounds like we're good to go.